BusinessLogicLayer/DTO/CurrencyRequestDto.cs

```diff
@@ -11,15 +12,17 @@ public class CurrencyRequestDto
  
         [JsonProperty("api_key")]
         public string api_key { get; set; }
+        [Required]
         [JsonProperty("from")]
         public string from { get; set; }
         
+        [Required]
         [JsonProperty("to")]
         public string to { get; set; }
-       
+        [Required(ErrorMessage = "You should write Integer number to convert")]
```

Comment:
You should write Integer number to convert -> Please provide integer value

Also why int. Shouldn't it be double or decimal?